Content.Server/Power/EntitySystems/ExtensionCableSystem.cs

```diff
@@ -248,6 +249,7 @@ private bool TryFindAvailableProvider(EntityUid owner, float range, [NotNullWhen
             var coordinates = xform.Coordinates;
             var nearbyEntities = grid.GetCellsInSquareArea(coordinates, (int) Math.Ceiling(range / grid.TileSize));
 
+            var candidates = new List<Tuple<float, ExtensionCableProviderComponent>>();
```

Comment:
Store a list of [value tuples](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples) instead. `Tuple<>` sucks.

Reply:
These are awesome, coming from Java-land these are awesome :o